.env.template

```diff
@@ -8,6 +8,7 @@ NEXT_PUBLIC_DISABLE_LOGIN_FOR_DEV=next-public-disable-login-for-dev
 
 NEXT_PUBLIC_CLIENT_ID=next-public-client-id
 NEXT_PUBLIC_SITE_NAME=next-public-site-name
+NEXT_PUBLIC_SITE_NAME_SHORT=next-public-site-name-short
```

Comment:
does it make sense to store short name (JJS, ER, etc.) as environment variable?
other option would be to carry suggested id pattern in original `questions` object.

Reply:
ya, ANY "[magic string](https://deviq.com/antipatterns/magic-strings)" should likely be moved to an env var